server/app/common/utils/server-utils.js

```diff
@@ -19,7 +19,7 @@ class ServerUtils {
       var exponentHex = exponentRaw.stream.hexDump(exponentStart, exponentEnd);
       var exponent = Hex.decode(exponentHex);
 
-      return { success: true, msg: { moduls: modulus, exponent: exponent } };
+      return { success: true, msg: { modulus: modulus, exponent: exponent } };
```

Comment:
e.g. here is functional change. I found no other mentioning of getRSAModulusAndExponent here [or entire github](https://github.com/search?q=getRSAModulusAndExponent&type=code) -- so may be it just was never used and could be just removed altogether?